src/db.c

```diff
@@ -253,6 +253,7 @@ int top_room;
 int top_shop;
 int top_repair;
 int top_vroom;
+int area_version;

```

Comment:
This is really only used in this file.  Probably should just be a local if it's not going to be used anywhere else.  But for now making it an extern in mud.h.